src/main/webapp/WEB-INF/view/activity.jsp

```diff
@@ -61,29 +63,41 @@ UserStore userStore = (UserStore) request.getAttribute("userStore");
   <ul>
     <%
        /* Used to format the time since the getCreationTime() method returns an instance, not a string. */
-       DateTimeFormatter formatter = DateTimeFormatter.ofLocalizedDateTime( FormatStyle.SHORT )
-         .withLocale( Locale.US )
-         .withZone( ZoneId.systemDefault() );
+      DateTimeFormatter formatter = DateTimeFormatter.ofLocalizedDateTime( FormatStyle.SHORT )
+        .withLocale( Locale.US )
+        .withZone( ZoneId.systemDefault() );
```

Comment:
We normally do NOT put extra spaces within the paranthesis, i.e., withZoen(ZoneId.systemDefault()).